### ISSUE TITLE
:recycle: refactor(navigation)

### ISSUE THREAD

**Issue by hirsch88** [baloise/design-system#784]
1. Implement active state for tabs 
2. Navigation updates
3. Accessibility
4. SEO
5. Performance
6. Tracking

---
Needs:
- #903 
- #905 
- #906 
- #907 
- #908 
- #909 
- #910  
- #911
---

## 1. Implement active state for tabs 

- Active states are visible on mobile and desktop
- heading has default, hover and active state
- menu-list-item has default, hover and active state

## 2. Navigation updates

- Meta Tabs, hide if there is only one available item

![image](https://user-images.githubusercontent.com/6384499/233358379-1f302ec8-a156-4d6b-a0a6-d69a262fa895.png)

- Scroll behaviour on desktop
  - Remember scroll position of the user and block scrolling in the background if the main navigation is open and is too big for the viewport
  
![image](https://user-images.githubusercontent.com/6384499/233358192-02616481-fec3-448a-8ad3-860eaabbed4a.png)

- Main navigation has enough space on the first line with the logo than render links directly after the logo
![Screenshot 2022-09-21 at 11 36 21](https://user-images.githubusercontent.com/17945926/191457302-a54545b8-4875-4686-bc55-ef34271f512c.png)

## 3. Accessibility

- All clickable navigation links should be accessible with the keyboard navigation (Tab key navigation)

![image](https://user-images.githubusercontent.com/6384499/233358796-28226513-cf86-4fbb-a294-47fbc6ad2163.png)

- Space or enter will open / activate the link. When keyboard focus is on tab or link with space block scrolling.

## 4. SEO

- Titles should not have h-tag, because there are rendered before the first h1
- Do not render all children pages in the navigation - as this eats up alle the link juice
- It is better only to render the children of the currently active page in the navigation

## 5. Performance

- Only render desktop or mobile navigation 
- Do not render all children pages in the navigation - less DOM nodes to render

![Image](https://user-images.githubusercontent.com/6384499/196690024-d9163552-0270-41af-ba38-c6dce3fa92f4.png)

<img width="755" alt="image" src="https://user-images.githubusercontent.com/6384499/195585549-25717766-f438-4b55-8b9b-4fed98efb816.png">

## 6. Tracking

- All links should trigger a google analytics event

**Comment by hirsch88** [comment]
@clastzoo add screenshots to 2.

**Comment by hirsch88** [comment]
Continues in #911
